pyrelational/models/mcdropout_model.py

```diff
@@ -42,6 +42,7 @@ def __call__(self, loader: DataLoader) -> torch.Tensor:
             raise ValueError("No current model, call 'train(train_loader, valid_loader)' to train the model first")
         predictions = []
         model = self.current_model
```

Comment:
As a generic question does model.eval() capture other things than gradients that should be in "evaluation" mode not caught by with torch.no_grad()?

Reply:
I think it handles dropout and batchnorms (and maybe other modules that behave differently during training and eval)